lib/spack/spack/cmd/find.py

```diff
@@ -342,10 +355,20 @@ def find(parser, args):
 
         count_suffix = " (not shown)"
         if not args.only_roots:
-            cmd.display_specs(results, args, decorator=decorator, all_headers=True)
             count_suffix = ""
+            kwargs = {"decorator": decorator, "all_headers": True, "status_fn": status_fn}
+            cmd.display_specs(results, args, **kwargs)
```

Comment:
I don't see why we need to change the invocation style here -- IMO this is clearer:

```suggestion
            kwargs = {"decorator": decorator, "all_headers": True, "status_fn": status_fn}
            cmd.display_specs(
                results,
                args,
                decorator=decorator,
                all_headers=True,
                status_fn=status_fn,
            )
```